### PR TITLE
updated get nearest premium function

### PR DESCRIPTION
**Pull request by Atmabuddy99**

### CODE REVIEW

File: src/fastbt/utils.py [hunk 1]
```diff
@@ -565,23 +565,23 @@ def get_nearest_premium(
...
+        return None
     
-    if isinstance(instrument_map,dict):  
+    elif isinstance(instrument_map,dict):  
         
-        difference=None
+        diff=None
         latest_symbol=None
         for symbols,ltp in instrument_map.items():
             last_trade_price=int(float(ltp))
```
Comment (uberdeveloper) [inline src/fastbt/utils.py:575]: why are you forcing this to be int

File: src/fastbt/utils.py
Comment (uberdeveloper) [inline src/fastbt/utils.py:1]: did you ran the tests on the new version

File: src/fastbt/utils.py (hunk 1 above)
Comment (Atmabuddy99) [reply src/fastbt/utils.py]: this test works in my system.Kindly check again if possible

Comment (uberdeveloper) [reply src/fastbt/utils.py]: this one passes

**Pull request merged**